src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,34 @@
+package duke.task;
+public class Todo {
+    private String name;
+    private boolean isDone = false;
+
+    public Todo(String name) {
+        setName(name);
+    }
+
+    public String getName() {
+
```

Comment:
This unexpected spacing seems to be inconsistent with the code style you followed in the rest of your codebase